5_People/career.md

```diff
@@ -6,7 +6,7 @@ Instead of lengthy and often biased per-person negotiations, Niteo uses a Salary
 
 We've done a lot of research before creating a system that works for us and big thanks go to companies who publish details about their own Salary Systems. These include [Basecamp](https://m.signalvnoise.com/how-we-pay-people-at-basecamp/), [Travis CI](https://www.youtube.com/watch?v=N8u9H6JDAzo), [Buffer](https://buffer.com/salary/data-scientist-2/average/), and others. We hope that by publishing ours, we inspire other companies to do the same.
 
-Salary System is a constant work in progress and we always strive to make it better and more fair for everyone in the company. The spreadsheet with all the data and calculations is available on `Boxcryptor/Niteans/5_People/Salary System`.
+Salary System is a constant work in progress and we always strive to make it better and more fair for everyone in the company. The spreadsheet with all the data and calculations is available on `Dropbox/Niteans/5_People/Salary System`.
```

Comment:
@zupo 2023 file needs to be included. There are only 2019 and 2020 files there.

Reply:
It was manual back then. It's in Scrooge and Salary Calculator now.

Reply:
@karantan How is that better than this?

![Screenshot 2024-01-25 at 18 11 11](https://github.com/teamniteo/handbook/assets/1309671/a228eec0-d673-403a-91f4-56802e81fee2)

Reply:
This calculation is wrong because the commitment bonus is rounded. Also, values from salary.com are changing on a daily basis so it is really hard to reverse engineer the logic without really looking into the code and understanding what exactly it's doing.
I was able to do this and confirmed that everything is correct. IMO if we have an open system I think every Nitean should need to be able to confirm that their calculation is correct, without looking at the code.

I don't think this is a lot of work. We need screenshots from salary.com graphs (so that we have a trace of which values have we taken for the base salary) and some sort of a .log file where the steps for the calculations are logged. Somethink like those [robot files](https://robotframework.org/#getting-started) except the content will be about calculation.

Before we do this we can also make an experiment and ask every nitean if they know how their salary is calculated manually.

Reply:
I would be strongly against adding another view, like the spreadsheet. The calculator is what needs to be readable, understandable and replicable. If it's not right now, we should definitely do it.

Reply:
1. Trace of values -> we have commits in the Salary Calc repo, so we do have a trace or log of changes. Would adding "as of <DATE>" next to "(base salary)" help?

2. rounding of commitment bonus -> it should be easy to remove the rounding from the Elm view code